lib/Fhp/MT940/Dialect/ComdirectMT940.php

```diff
@@ -0,0 +1,268 @@
+<?php
```

Comment:
Die Datei gehört in einen separaten PR oder ganz raus -- momentan wird die Klasse nicht verwendet.